detections/endpoint/RMM_Agent_Install_Followed_By_First_Outbound_Connection.yml

```diff
@@ -0,0 +1,117 @@
+name: RMM Agent Install Followed By First Outbound Connection
+id: null
+version: 1
+date: '2025-08-17'
+author: Riyan Pratama (sudo3rs)
+data_source:
+  - Sysmon
+  - Windows Security Event Log
+  - Windows System Event Log
+  - Web Proxy
+  - Firewall
+type: TTP
+status: production
+description: |
+  Detects installation or service registration of common Remote Monitoring & Management (RMM) tools
+  (e.g., AnyDesk, ConnectWise ScreenConnect, Atera, Splashtop, TeamViewer) followed within a short
+  window by the first outbound connection from the same host to the vendor’s cloud endpoints.
+  This correlation highlights “living-off-RMM” abuse where an attacker deploys a legitimate RMM
+  agent for persistent remote access, then immediately dials out to external control infrastructure.
+  A SOC should investigate promptly, as this often indicates hands-on-keyboard activity enabling
+  remote control, data staging/exfiltration, or further lateral movement.
+search: |
+  (
+    (`wineventlog_system` EventCode=7045) OR
+    (`wineventlog_security` EventCode=4697) OR
+    (`sysmon` EventCode=1)
+  )
+  (Image="*anydesk*.exe" OR Image="*screenconnect*" OR Image="*connectwise*" OR Image="*atera*" OR Image="*splashtop*" OR Image="*teamviewer*"
+   OR ServiceFileName="*AnyDesk*" OR ServiceFileName="*ScreenConnect*" OR ServiceFileName="*ConnectWise*" OR ServiceFileName="*Atera*" OR ServiceFileName="*Splashtop*" OR ServiceFileName="*TeamViewer*")
+  | eval indicator=coalesce(Image, ServiceFileName, ServiceName)
+  | eval rmm_brand=case(
+      like(lower(indicator), "%anydesk%"), "AnyDesk",
+      like(lower(indicator), "%screenconnect%") OR like(lower(indicator), "%connectwise%"), "ScreenConnect",
+      like(lower(indicator), "%atera%"), "Atera",
+      like(lower(indicator), "%splashtop%"), "Splashtop",
+      like(lower(indicator), "%teamviewer%"), "TeamViewer",
+      true(), "Other"
+    )
+  | eval marker="install"
+  | table _time host user Image ServiceName ServiceFileName rmm_brand marker
+  | append [
+      search (index=proxy OR index=fw OR index=network)
+        (
+          dest_domain="*.anydesk.com" OR dest_domain="*.anydeskcontrol.com" OR
+          dest_domain="*.screenconnect.com" OR dest_domain="*.connectwise.com" OR
+          dest_domain="*.atera.com" OR dest_domain="*.splashtop.com" OR
+          dest_domain="*.teamviewer.com"
+        )
+        OR (app IN ("anydesk","connectwise","atera","splashtop","teamviewer"))
+      | eval marker="connect"
+      | table _time host dest dest_ip dest_domain app marker
+    ]
```

Comment:
Most if not all installation of an RMM agent will lead to a network connection to the vendor domain for various downloads and check. 

For example anydesk will contact `boot.net.anydesk.com`.

For this the rule will trigger basically on any installation not just the first connection.

I suggest you add further conditions to strengthen the logic.